docs/books/lxd_server/08-snapshots.md

```diff
@@ -2,38 +2,38 @@
 title: 8 Container Snapshots
 author: Steven Spencer
 contributors: Ezequiel Bruni
-tested with: 8.5, 8.6, 9.0
+tested with: 8.8, 9.2
 tags:
   - lxd
   - enterprise
   - lxd snapshots
 ---
 
-# Chapter 8: Container Snapshots
+# Chapter 8: container snapshots
 
-Throughout this chapter you will need to execute commands as your unprivileged user ("lxdadmin" if you've been following along from the beginning of this book).
+Throughout this chapter you will need to run commands as your unprivileged user ("lxdadmin" if you've been following along from the beginning of this book).
 
-Container snapshots, along with a snapshot server (which we will get to more later), are probably the most important aspect of running a production LXD server. Snapshots ensure quick recovery, and can be used for safety when you are, say, updating the primary software that runs on a particular container. If something happens during the update that breaks that application, you simply restore the snapshot and you are back up and running with only a few seconds worth of downtime.
+Container snapshots, along with a snapshot server (more on that later), are probably the most important aspect of running a production LXD server. Snapshots ensure quick recovery. It is a good idea to use them as a fail safe when updating the primary software that runs on a particular container. If something happens during the update that breaks that application, you just restore the snapshot and you are back up and running with only a few seconds worth of downtime.
 
-The author used LXD containers for PowerDNS public facing servers, and the process of updating those applications became so much more worry-free, since you can snapshot the container first before continuing.
+The author used LXD containers for PowerDNS public facing servers, and the process of updating those applications became less worrisome, because of taking a snapshot of the container first before continuing.
```

Comment:
"because of taking a snapshot of the container first before continuing" could be more simply phrased as "thanks to taking snapshots before every update"